reports/Michigan.md

```diff
@@ -163,6 +163,20 @@ id: mi-grandrapids-2
 * https://twitter.com/sparrowsongs_/status/1266948928966139904
 
 
+### Police assault peaceful protesters, among them Breonna Taylor's family members | July 12th
+
+City of Grand Rapids Police Department officers seen pre-emptively pushing, pulling, and shoving peaceful protesters, as they circle around an officer arresting or detaining a man.
+
+The detaining officer appears to force the man to his knees before the other officers finish encircling and view of the man is lost.
+
+Two different women, each in different videos, are shown falling or fallen over as a result of the shoving and pulling.
+
+**Links**
+* https://mobile.twitter.com/OfficAlyNicole/status/1283181085577420810
```

Comment:
This is my fault for sending on my phone, but can you change this to the desktop link?